tests/config/nextflow.config

```diff
@@ -6,34 +6,75 @@ params {
 }
 
 process {
-    cpus = 2
-    memory = 3.GB
-    time = 2.h
+    resourceLimits = [
+        cpus: 2,
+        memory: '4.GB',
+        time: '1.h'
+    ]
 }
 
-if ("$PROFILE" == "singularity") {
```

Comment:
Will this still work for pytest?

Reply:
not sure...I can leave it in there but, I think it should work

Reply:
but, feel free to add changes and merge this